src/middlewares/authorization.js

```diff
@@ -0,0 +1,17 @@
+import verifyToken from '../utils/authentication';
+
+const authentication = (req, res, next) => {
+  const bearerHeader = req.headers['x-access-token'] || req.query.token || req.headers.authorization;
```

Comment:
Please change this variable name to `token` since we are not using `Bearer` for authentication.